syz-manager/manager.go

```diff
@@ -88,6 +88,8 @@ type Manager struct {
 	// For checking that files that we are using are not changing under us.
 	// Maps file name to modification time.
 	usedFiles map[string]time.Time
+
+	kcovFilter *CoverFilter
```

Comment:
We never use term "kcov" as of now. KCOV is a Linux-specific interface. This is OS-agnostic code. We generally refer to coverage as "coverage" here. And you call it _coverage_ filter even in executor.
Rename it to `coverFilter` for consistency.